Simplified/ReportIssue.storyboard

```diff
@@ -77,6 +77,9 @@
                     </navigationItem>
                     <nil key="simulatedTopBarMetrics"/>
                     <nil key="simulatedBottomBarMetrics"/>
+                    <connections>
+                        <outlet property="tableView" destination="GRj-EQ-BXo" id="8Y9-oc-jsc"/>
+                    </connections>
```

Comment:
Yeah you're right. You can then get rid of the line that re-initalizes the tableview in viewDidLoad()